build-tests/install-test-workspace/build.js

```diff
@@ -139,11 +139,7 @@ const pnpmInstallArgs = [
   '--recursive',
   '--link-workspace-packages=false',
   // PNPM gets confused by the rewriting performed by our .pnpmfile.cjs afterAllResolved hook
-  '--frozen-lockfile=false',
-
-  // Workaround for this issue:
-  // https://github.com/pnpm/pnpm/issues/6778#issuecomment-1762418094
-  '--config.confirmModulesPurge=false'
+  '--frozen-lockfile=false'
```

Comment:
Seems like you inadvertently reverted PR #4393

Reply:
undid this, thank you Pete